packages/fast-figma-plugin-msft/src/core/ui/components/stealth-button/index.ts

```diff
@@ -0,0 +1,119 @@
+import { css, customElement, FastElement, html } from "@microsoft/fast-element";
+import { designSystemConsumer } from "@microsoft/fast-components/dist/design-system-consumer";
+import {
+    accentForegroundCutRest,
+    neutralFillStealthActive,
+    neutralFillStealthHover,
+    neutralFillStealthRest,
+} from "@microsoft/fast-components/dist/styles/recipes";
+
+const template = html`
+    <button>
+        <slot id="glyph" name="glyph"></slot>
+        <span><slot id="content"></slot></span>
+    </button>
+`;
+
+const styles = css`
+    :host {
+        display: inline-block;
+        outline: none;
+    }
+
+    :host(.has-glyph.has-content) span {
+        margin-inline-start: 8px;
+    }
+
+    button {
+        box-sizing: border-box;
+        padding: 0 8px;
+        min-width: 32px;
+        height: 32px;
+        border: none;
+        border-radius: calc(var(--corner-radius) * 1px);
+        background: var(--neutral-fill-stealth-rest);
+        fill: var(--neutral-foreground-rest);
+        outline: none;
+        font-size: inherit;
+    }
+
+    button:focus {
+        border-color: var(--neutral-outline-rest);
+    }
+    button:hover {
+        background: var(--neutral-fill-stealth-hover);
+    }
+
+    button:active {
+        background: var(--neutral-fill-stealth-active);
+    }
+
+    :host([aria-expanded="true"]) button {
+        background: #18a0fb;
+        fill: var(--accent-foreground-cut-rest);
+    }
+`;
+
+@customElement({
+    name: "td-stealth-button",
+    template,
+    styles,
+    shadowOptions: {
+        mode: "open",
+        delegatesFocus: true,
+    },
+})
+@designSystemConsumer({
+    recipes: [
+        neutralFillStealthRest,
+        neutralFillStealthHover,
+        neutralFillStealthActive,
+        accentForegroundCutRest,
+    ],
+})
+export class StealthButton extends FastElement {
```

Comment:
would this be deprecated in the future to use a stealth button from fast-components in the future?

Reply:
Yes